src/test/java/pl/wedding/management/service/user/InMemoryRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package pl.wedding.management.service.user;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.wedding.management.service.user.model.EMail;
+import pl.wedding.management.service.user.model.User;
+
+import static org.assertj.core.api.Assertions.*;
+
+class InMemoryRepositoryTest {
+
+    private static final char[] DUMMY_PASSWORD = new char[]{'p', 'a', 's', 's', 'w', 'o', 'r', 'd'};
```

Comment:
How about simpler? 
```suggestion
    private static final char[] DUMMY_PASSWORD = "password".toCharArray();
```

Reply:
I am not sure if there is a need to add this "password" to string literal.

Reply:
it's readable. But it's not easy to change to something more complex.

Reply:
I do not see any reason to change this to something more complex in this place, do you?

Reply:
There's never the reason at the beginning. That's not the reason to complicate further work in order to having to work longer.

Reply:
I am not sure what you are trying to say. 
1) Type of this constant is char array so why should we create some String and then casting it again to char array by native method instead of simple creating char array?
2) Current approach  is still readable, isn't it?
3) Would you have some issues with changing value of this constant? Changing value in String will be about 1,5 second faster. It would not be a time saver.